src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -127,23 +127,15 @@ export const portfolio = createSlice({
       // add the `action.meta[SHOULD_AUTOBATCH]` field the enhancer needs
       prepare: prepareAutoBatched<WalletId>(),
     },
-
-    upsertPortfolio: {
-      reducer: (draftState, { payload }: { payload: Portfolio }) => {
-        // upsert all
-        draftState.accounts.byId = merge(draftState.accounts.byId, payload.accounts.byId)
-        draftState.accounts.ids = Object.keys(draftState.accounts.byId)
-
-        draftState.accountBalances.byId = merge(
-          draftState.accountBalances.byId,
-          payload.accountBalances.byId,
-        )
-        draftState.accountBalances.ids = Object.keys(draftState.accountBalances.byId)
-      },
-
-      // Use the `prepareAutoBatched` utility to automatically
-      // add the `action.meta[SHOULD_AUTOBATCH]` field the enhancer needs
-      prepare: prepareAutoBatched<Portfolio>(),
+    upsertPortfolio: (draftState, { payload }: { payload: Portfolio }) => {
```

Comment:
Removed `prepare` and consequently `reducer`, which isn't possible without `prepare` (at least at types-level)